packages/data/src/registry.js

```diff
@@ -17,11 +17,21 @@ import createCoreDataStore from './store';
  *
  * @typedef {WPDataRegistry}
  *
- * @property {Function} registerGenericStore
- * @property {Function} registerStore
- * @property {Function} subscribe
- * @property {Function} select
- * @property {Function} dispatch
+ * @property {Function} registerGenericStore Given a namespace key and settings
```

Comment:
I'd suggest making it the standard for this entire project (and all of WordPress, if possible) to use the tag `@prop` instead of `@property` to save on precious line lengths.

It'll help somewhat with the smushed descriptions.

Reply:
>It'll help somewhat with the smushed descriptions.

In some respect I appreciate the constraints to encourage being brief with the names and descriptions.

I don't have a strong feeling about `@property` vs. `@prop`, but it needs to be made in the documentation standards, which [currently lists `@prop` as unsupported](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#unsupported-jsdoc-tags). A topic for a future JavaScript meeting, perhaps?